PURL-TYPES.rst

```diff
@@ -81,15 +81,24 @@ conan
 -----
 ``conan`` for Conan C/C++ packages:
 
-- The default repository is ``https://center.conan.io``
-- The ``namespace`` is the user if present
-- The ``name`` is the package name.
-- The ``version`` is the package version.
-- The qualifier ``channel`` must be not empty if namespace is present
+- The default repository is ``https://center.conan.io``.
+- The ``namespace`` is the vendor.
+- The ``name`` is the Conan package name.
+- The ``version`` is the Conan package version.
+- The qualifier ``channel`` is the Conan channel, it is required if the Conan package was published with channel.
```

Comment:
"with channel" -> "with *a* channel" / "to a channel"?

Reply:
i could not come up with a clear formulation for this, so now i have a new approach where i introduce the Conan package reference scheme so i can refer to the tokens in it. Does that work for you?